common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/ExternalShuffleClient.java

```diff
@@ -49,6 +49,7 @@
   private final TransportConf conf;
   private final boolean authEnabled;
   private final SecretKeyHolder secretKeyHolder;
+  private final long registrationTimeoutMilli;
```

Comment:
"MS" or "Millis" is more consistent. Milli suggests something different. https://en.wikipedia.org/wiki/Milli_Vanilli